ui/StatusQ/src/StatusQ/Components/StatusRoundedMedia.qml

```diff
@@ -112,31 +126,52 @@ StatusRoundedComponent {
         }
     }
 
-    Loader {
-        id: mediaLoader
-        anchors.fill: parent
-        asynchronous: true
-        visible: !root.isError && !root.isLoading
+    implicitWidth: {
+        // Use Painted width so that the rectangle follow width of the image actually painted
+        if(!!mediaLoader.item && (mediaLoader.item.paintedWidth > 0 || mediaLoader.item.paintedHeight > 0)) {
```

Comment:
Why also check for `paintedHeight`? This will return values smaller or equal with 0 that are of no use for sizing. Same for the `implicitHeight` binding bellow.

Reply:
Hey Stefan, the thing here is that I need the rectangle holding the image to follow the dimensions of the actually drawn image. SO say I set width and ehight of image to 240 by 240 but in order to respect the Image.PreserveAspectRatio, the image drawn will actually be 112 by 240 (for example). I need the parent rectangle to have size 112 by 240 as well and hence this logic.